src/test/java/com/wikia/webdriver/elements/fandom/components/HeroBlock.java

```diff
@@ -16,7 +16,7 @@ public boolean isDisplayed() {
     try {
       return heroMosaic.isDisplayed();
     } catch (ElementNotFoundException e) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS1166)